### PR TITLE
Documented RKE2 feature flag and new RKE2 options for Windows clusters 

### LINKED ISSUE #3497: [Rancher2] Document RKE2 feature flag and new RKE2 options for Windows clusters
https://github.com/rancher/dashboard/issues/3222

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3497 

See also https://github.com/rancher/dashboard/issues/3222

### CODE REVIEW

File: content/rancher/v2.6/en/installation/resources/feature-flags/_index.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ For example, if you install Rancher, then set a feature flag to true with the Ra
 
 The following is a list of the feature flags available in Rancher:
 
+- `rke2`: We have introduced the ability to provision RKE2 clusters as tech preview. By default, this feature flag is enabled, which allows users to attempt to provision these type of clusters. Additionally, users can now select "Windows" in the Cluster Management page, then choose the appropriate options to configure the custom Windows cluster. Only Windows worker nodes may be registered.
```
Comment (btat) [inline content/rancher/v2.6/en/installation/resources/feature-flags/_index.md:28]:
```suggestion
- `rke2`: We have introduced the ability to provision RKE2 clusters as tech preview. By default, this feature flag is enabled, which allows users to attempt to provision these type of clusters.
```

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 2]
```diff
@@ -34,13 +34,16 @@ _Tech Preview_
...
+The RKE2 provisioning tech preview also includes installing RKE2 on Windows clusters. When provisioning a new Windows cluster, users can select "Windows" and then choose the appropriate options to configure the custom cluster under the Cluster Management page. Only Windows worker nodes may be registered.
+
+Windows features for RKE2 include:
 
 - Windows Containers with RKE2 powered by containerd
 - Added provisioning of Windows RKE2 custom clusters directly from the Rancher UI
 - Calico CNI for Windows RKE2 custom clusters
 - SAC releases of Windows Server (2004 and 20H2) are included in the technical preview
 
+
```
Comment (btat) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:46]: Extra new line is not needed

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:46]: Removed line.

File: content/rancher/v2.6/en/installation/resources/feature-flags/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.6/en/installation/resources/feature-flags/_index.md:28]: Removed sentence.

**Pull request merged**